src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.split(';');
+  const stylesObject = {};
+
+  declarations.forEach((declaration) => {
+    const [property, value] = declaration.split(':').map((part) => part.trim());
```

Comment:
The `map` function is used correctly to trim the parts of the declaration. However, consider using destructuring with default values to handle potential undefined values that could occur if the input string is malformed.